scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -368,35 +501,39 @@ private static CommandLine getCommandLine(String[] args, String loginFilePath, S
         return cmd;
     }
 
-    private static void updateUserGroups(String login, String[] groups, Multimap<String, String> groupsMap) {
+    private static void updateUserGroups(String login, Collection<String> groups, Multimap<String, String> groupsMap) {
         groupsMap.removeAll(login);
         for (String group : groups) {
-            groupsMap.put(login, group);
-            System.out.println("Added group " + group + " to user " + login);
+            if (!group.isEmpty()) {
+                groupsMap.put(login, group);
+                System.out.println("Adding group " + group + " to user " + login);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)